src/main/webapp/scripts/view.js

```diff
@@ -0,0 +1,36 @@
+export { authorizeButton, signoutButton, commentButton, videoId, commentElement, resultContainer };
+export { successfulApiCallback, failedApiCallback, updateSigninStatus }
+
+function updateDom(text, containerDOM) {
+    containerDOM.innerHTML = '';
+    containerDOM.innerText = text;
+}
+
+const authorizeButton = document.getElementById('authorize-button');
+const signoutButton = document.getElementById('signout-button');
+const commentButton = document.getElementById('comment-button');
+
+const videoId = "gjKpAAezWQA";
+const commentElement = document.getElementById("comment-text");
+const resultContainer = document.getElementById('result-text');
+
+function updateSigninStatus(isSignedIn) {
+    if (isSignedIn) {
+        authorizeButton.style.display = 'none';
+        signoutButton.style.display = 'block';
+        commentButton.style.display = 'block';
+    } else {
+        authorizeButton.style.display = 'block';
+        signoutButton.style.display = 'none';
+        commentButton.style.display = 'none';
```

Comment:
I did something similar in my starter project with changing the style in JS files. It is not recommended, so maybe you could do something like change the button function instead? like 
```
const authButton = document.getElementById('auth-button');
if(isSignedIn) {
  authButton.onclick = handleSignoutClick;
  addCommentButton() //adds a comment button to the page
} else {
  authButton.onclick = handleAuthClick;
}
```
I'm not sure if this is the right way to avoid changing the style in JS, I'm sure there are many options that you can choose from

Reply:
Done